src/grid/boundary_conditions.cpp

```diff
@@ -151,12 +151,15 @@ int Grid3D::Check_Custom_Boundary(int *flags, struct parameters P)
   }
 
   for (int i = 0; i < 6; i++) {
+    /* Alwin: I am disabling this check because it is needlessly occurring every timestep.
```

Comment:
is this disabled always? Is the check occurring somewhere else? 

- I think it should happen somewhere, probably in the error checking function.
- If we don't want it here then it should be deleted, not commented out

Reply:
Fixed in [297225b](https://github.com/cholla-hydro/cholla/pull/315/commits/297225bd5c6d786d278ac652adf14a8c5901a5bd)